src/test/scala/cromwell/HelloWorldActorSpec.scala

```diff
@@ -47,25 +47,23 @@ class HelloWorldActorSpec extends CromwellSpec(ActorSystem("HelloWorldActorSpec"
   val TestExecutionTimeout = 5000 milliseconds
 
   "A WorkflowActor" should {
-
     "start" in {
+      val binding = WdlBinding.process(HelloWorld.WdlSource)
+      val coercedInputs = binding.coerceRawInputs(HelloWorld.RawInputs).get
+      /*
+        The TestFSMRef is kind of quirky, defining it here instead of the buildWorkflowActor function. It could
+        be generalized a bit but it is probably not worth the hassle for a test class
+       */
+      val fsm = TestFSMRef(new WorkflowActor(UUID.randomUUID(), binding, coercedInputs, new LocalBackend))
+      assert(fsm.stateName == WorkflowSubmitted)
+      fsm ! Start
```

Comment:
My concern was that you do that now via direct subscription and if it was already running there's no guarantee that it hasn't already gone through the states. What didn't occur to me is that you could subscribe before starting it.